src/pointer-analysis/value_set.h

```diff
@@ -309,7 +309,7 @@ class value_sett
   /// Clears value set (not used in the CBMC repository)
   void make_any()
   {
-    values.clear();
+    clear();
   }
```

Comment:
Hmm: What's the point of this method when it (now) does exactly the same as `clear()`? The comment really isn't helpful, because it says what the other method obviously does... The git history doesn't help, because it predates git... @kroening and @smowton Could you please figure this out?

Reply:
I'd been assuming out-of-tree users, but @kroening shall I just erase?

Reply:
Actually it would be trivial for such people to switch to `clear()`, I'll go ahead and do this